docs/technical_reference/unit_models/uv_aop.rst

```diff
@@ -50,41 +50,54 @@ The UV system includes the state variables from the associated property package,
-   "Overall pseudo-first order rate constant", ":math:`k_0`", "rate_constant", "['Liq', j]", ":math:`\text{s}^{-1}`"
-   "Pseudo-first order rate constant for direct photolysis of component", ":math:`k_d`", "photolysis_rate_constant", "['Liq', j]", ":math:`\text{s}^{-1}`"
-   "Pseudo-first order rate constant for indirect photolysis of component", ":math:`k_i`", "reaction_rate_constant", "['Liq', j]", ":math:`\text{s}^{-1}`"
+   "Inactivation rate coefficient", ":math:`k`", "inactivation_rate", "[p, j]", ":math:`\text{m}^2\text{/J}`"
+   "Overall pseudo-first order rate constant", ":math:`k_0`", "rate_constant", "[p, j]", ":math:`\text{s}^{-1}`"
+   "Pseudo-first order rate constant for direct photolysis of component", ":math:`k_d`", "photolysis_rate_constant", "[p, j]", ":math:`\text{s}^{-1}`"
+   "Pseudo-first order rate constant for indirect photolysis of component", ":math:`k_i`", "reaction_rate_constant", "[p, j]", ":math:`\text{s}^{-1}`"
    "UV dose", ":math:`D`", "uv_dose", None, ":math:`\text{J/}\text{m}^2`"
    "Average intensity of UV light", ":math:`I`", "uv_intensity", None, ":math:`\text{J/}\text{m}^2\text{/s}`"
    "Exposure time of UV light", ":math:`t`", "exposure_time", None, ":math:`\text{s}`"
-   "Electricity demand of components", ":math:`E_j`", "electricity_demand_phase_comp", "[t, 'Liq', j]", ":math:`\text{W}`"
-   "Electricity efficiency per log order reduction (EE/O)", ":math:`EE/O_j`", "electrical_efficiency_phase_comp", "[t, 'Liq', j]", ":math:`\text{J/}\text{m}^3`"
+   "Electricity demand of components with phase", ":math:`E_{p, j}`", "electricity_demand_phase_comp", "[t, p, j]", ":math:`\text{W}`"
+   "Electricity demand of components", ":math:`E_j`", "electricity_demand_comp", "[t, j]", ":math:`\text{W}`"
+   "Electricity demand", ":math:`E`", "electricity_demand", "[t]", ":math:`\text{W}`"
+   "Electricity efficiency per log order reduction (EE/O)", ":math:`EE/O_{p, j}`", "electrical_efficiency_phase_comp", "[t, p, j]", ":math:`\text{J/}\text{m}^3`"
    "Lamp efficiency", ":math:`\eta`", "lamp_efficiency", None, None
-   "UV transmittance", ":math:`UVT`", "UVT", None, None
-   "UV absorbance", ":math:`UVA`", "UVA", None, None
 
 **Users must provide values for and 'fix' certain variables to solve the model with DOF=0. Thus, users should fix**
     * either 'inactivation_rate' or 'rate_constant',
     * either 'photolysis_rate_constant' or 'reaction_rate_constant',
     * two variables out of 'uv_dose', 'uv_intensity' and 'exposure_time',
     * either 'electricity_demand_phase_comp' or 'electrical_efficiency',
-    * either 'UVT' or 'UVA',
     * and 'lamp_efficiency'.
 
 **However, users may later unfix certain variables for optimization purposes.**
 
+When setting the configuration options to calculate the UV dose, the 'uv_dose' variable is no longer specified with either 'uv_intensity' and 'exposure_time' specified.
```

Comment:
What are the configuration options now? Moreover, I don't think you need to explain what's new vs. what is no longer existent in the model. These are good notes for our team to understand the changes, but not necessarily helpful for users that likely didn't get a chance to see the "old version."

Reply:
I've clarified the `uv_dose_type` configuration for ease of users to understand.